ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -464,7 +465,8 @@ export class OntotextYasguiWebComponent {
     this.getOntotextYasgui()
       .then((ontotextYasgui) => {
         ontotextYasgui.refresh();
-        VisualisationUtils.setYasqeFullHeight(this.renderingMode, VisualisationUtils.resolveOrientation(this.isVerticalOrientation));
+        const editorHeight = ontotextYasgui.getTab(ontotextYasgui.getTabId()).persistentJson.yasqe.editorHeight;
```

Comment:
This can be moved in OntotextYasgui class as function that can be used everywhere.